.github/workflows/check_transport.yml

```diff
@@ -48,11 +49,44 @@ jobs:
             transport: native
     steps:
       - uses: actions/checkout@v3
+        with:
+          path: reactor-netty
+          fetch-depth: 0 #needed by spotless
+      - uses: actions/checkout@v3
+        with:
+          repository: netty/netty
+          ref: main
+          path: netty
+      - uses: actions/checkout@v3
+        with:
+          repository: netty-contrib/codec-haproxy
+          path: codec-haproxy
+      - uses: actions/checkout@v3
+        with:
+          repository: netty-contrib/codec-extras
+          path: codec-extras
+      - uses: actions/checkout@v3
+        with:
+          repository: netty-contrib/socks-proxy
+          path: socks-proxy
       - uses: gradle/wrapper-validation-action@v1
-      - name: Set up JDK 1.8
+      - name: Set up JDK 17
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 17
+      - name: Build netty
+        run: ./mvnw install -pl :netty-buffer,:netty5-handler,:netty5-resolver-dns,:netty5-resolver-dns-classes-macos,:netty5-transport-classes-epoll,:netty5-transport-classes-kqueue,:netty5-codec-http,:netty5-codec-http2 -DskipTests=true
+        working-directory: ./netty
```

Comment:
isn't the directory name that must be passed as arguments to the "./mvnw install -pl" option (instead of the artifact ids) ?

I mean: instead of : 
```
        run: ./mvnw install -pl :netty-buffer,:netty5-handler,... -DskipTests=true
```   
then:
```
        run: ./mvnw install -pl :buffer,:handler,... -DskipTests=true
```

Reply:
these were artifact ids, however I removed this because we have Netty 5 snapshots on sonatype